lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js

```diff
@@ -50,8 +62,11 @@ export default function baseRenderer(monitoredTrip) {
     )
   }
 
-  // set some alert data if the matching itinerary exists
-  data.alerts = data.matchingItinerary && data.matchingItinerary.alerts
+  // Set some alert data if the matching itinerary exists.
+  // (Convert alert timestamps in milliseconds to seconds for display using alerts body.)
+  data.alerts =
+    data.matchingItinerary &&
+    data.matchingItinerary.alerts.map(withAlertTimestampsInSeconds)
```

Comment:
could you write this using null coalescing operator?

Reply:
Did you mean optional chaining? As written, `data.alerts` can be `null` and the existing code supports that.